packages/apollo-cli/src/load-schema.ts

```diff
@@ -17,7 +17,7 @@ export async function loadSchema(
 
   if (dependency.endpoint && dependency.endpoint.url) {
     try {
-      return await fetchSchema(dependency.endpoint, config.projectFolder);
+      return await fetchSchema(dependency.endpoint, config.projectFolder, dependency.skipsSSLValidation);
```

Comment:
If we put `skipSSLValidation` under `endpoint` we can avoid passing in an extra parameter.

Reply:
Done